coalib/processes/Processing.py

```diff
@@ -344,7 +347,35 @@ def instantiate_processes(section,
         log_printer,
         ignored_file_paths=glob_list(section.get('ignore', "")),
         limit_file_paths=glob_list(section.get('limit_files', "")))
+
+    # This stores all matched files irrespective of whether coala is run
+    # only on changed files or not. Global bears require all the files
+    complete_filename_list = filename_list
+
+    # Start tracking all the files
+    if cache:
```

Comment:
I'm sorry, but I don't remember if we had this discussion :( (I do have a really bad memory though)
What is this about ?